app/lib/features/comments/actions/sbumit_comment.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger('a3::submit::comment');
+
+Future<void> submitComment(
+  BuildContext context,
+  String plainDescription,
+  String htmlBodyDescription,
+  CommentsManager manager,
+) async {
+  final lang = L10n.of(context);
+  if (plainDescription.isEmpty) {
+    EasyLoading.showToast(lang.youNeedToEnterAComment);
+    return;
+  }
+  EasyLoading.show(status: lang.submittingComment);
+  try {
+    final draft = manager.commentDraft();
+    draft.contentFormatted(plainDescription, htmlBodyDescription);
+    await draft.send();
+    FocusManager.instance.primaryFocus?.unfocus();
+    if (!context.mounted) {
+      EasyLoading.dismiss();
+      return;
+    }
+    EasyLoading.showToast(lang.commentSubmitted);
```

Comment:
we should indicate whether we were successful so the outer code can act accordingly (see below).
```suggestion
    await draft.send();
    FocusManager.instance.primaryFocus?.unfocus();
    EasyLoading.showToast(lang.commentSubmitted);
    return true;
```